_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -27343,6 +27346,18 @@
 "jVw" = (
 /turf/open/floor/iron/white,
 /area/station/medical/medbay/central)
+"jVQ" = (
+/obj/structure/industrial_lift/tram{
+	icon_state = "titanium_white"
+	},
+/obj/structure/window/reinforced/shuttle/tram,
+/obj/machinery/computer/tram_controls{
+	icon_keyboard = null;
+	icon_state = "tram_controls";
+	dir = 1
+	},
```

Comment:
```suggestion
/obj/machinery/computer/tram_controls{
	icon_state = "tram_controls";
	dir = 1
	},
```

![image](https://user-images.githubusercontent.com/34697715/208605571-9df187f7-e36d-4405-a13f-15c45f234f72.png)

I also don't really like the fact that we're deciding the icon_state map-side, this _will_ break things in the future if people decide to change the DMI naming scheme. Would be best to make a subtype to ensure we can catch it code-side at least.

Reply:
I will process all this and revise.

Reply:
So I'm moving the console back to the middle because I don't want to hold up the PR while learning directional helpers.

icon_keyboard = null is used, see the difference when it's removed, it tries to put a keyboard on the tram controls

![image](https://user-images.githubusercontent.com/83487515/208767178-c5ccddd6-90b2-42b9-b59a-6fd35983e71e.png)
![image](https://user-images.githubusercontent.com/83487515/208767265-ea0f39bf-a371-425e-aa28-60b3eef28884.png)

Reply:
I'll do the window mounted tram controls in a separate PR.

Reply:
directional helpers aren't that hard tbh, it's just a macro that spins out already pre-fabricated types so you can use it in a map editor (ever do stuff where you see something like `/obj/structure/sign/no_smoking/directional/north`?). it's your call though

Reply:
Yeah, I started to look. I'm going to be busy for a like a week, so I'll just make a new PR for the tram controls when I'm free.

Reply:
yeah no worries